### PR TITLE
add import tags in list of more opt

### PR DESCRIPTION
**Pull request by steve02081504**

### CODE REVIEW

File: public/scripts/power-user.js [hunk 1]
```diff
@@ -282,6 +282,7 @@ let power_user = {
     forbid_external_media: true,
     external_media_allowed_overrides: [],
     external_media_forbidden_overrides: [],
+    import_card_tags: true,
```
Comment (Cohee1207) [inline public/scripts/power-user.js]: No. Card tags should be an opt-in.

Comment (steve02081504) [reply public/scripts/power-user.js]: But this option just makes a window pop up when importing characters. The user can choose to cancel it.

Comment (Cohee1207) [reply public/scripts/power-user.js]: This is annoying and should be left as an option. Every creator has their own vision of tags, including capitalization, spelling, etc. This should not be on by default.

Comment (steve02081504) [reply public/scripts/power-user.js]: got it, fixed

**Pull request merged**